Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -659,6 +679,17 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
             },
 
             /**
+             * The identifier of the default visible method
+             *
+             * @attribute defaultVisibleMethod
+             * @type String
+             * @default 'finder'
+             */
+            defaultVisibleMethod: {
```

Comment:
you should not introduce an attribute for that. First because, we don't want that to be settable externally and second because the default value is already expressed in `visibleMethod` attribute